submodules/LIB/hardware/syn/Makefile

```diff
@@ -28,6 +31,10 @@ OUTPUT_DIR=results
 #default node
 NODE ?= umc130
 
+#open PDK NODE
+Open_NODE ?= sky130
+Open_hash ?= 6d4d11780c40b20ee63cc98e645307a9bf2b2ab8
```

Comment:
what is this ?

Reply:
is there a tag? a version? why do we need to know a hash??

Reply:
The open node is the pdk to this day exist 2 gf180 and sky130, but the hash it is not required.